content/asset-tracking/using-the-sdks.textile

```diff
@@ -86,7 +86,8 @@ The required methods are:
 The optional methods are:
 
 - profile := Called to set the means of transport being used for the initial state of publishers created from this builder. If not set then the default value is @RoutingProfile.DRIVING@.
-- context := Called to provide the Android runtime context (on Android only).
+- locationSource := Sets the location source to be used instead of the GPS. The location source provides location updates for the @Publisher@.
+- androidContext := Called to provide the Android runtime context (on Android only).
```

Comment:
The name is correct now however this thing is required for Android but here it's under "The optional methods" wink

Reply:
Fixed. `androidContext` moved to required.